model/project_parser.go

```diff
@@ -567,10 +567,13 @@ func LoadProjectInto(ctx context.Context, data []byte, opts GetProjectOpts, iden
 	// return intermediateProject even if we run into issues to show merge progress
 	for _, path := range intermediateProject.Include {
 		opts.RemotePath = path.FileName
-		if opts.PatchOpts.patch != nil {
-			if opts.PatchOpts.patch.ConfigChanged(path.FileName) {
+		// read from the patch diff if a change has been made in any of the include files
+		if opts.ReadFileFrom == ReadFromPatch || opts.ReadFileFrom == ReadFromPatchDiff {
+			if opts.PatchOpts.patch != nil && opts.PatchOpts.patch.ConfigChanged(path.FileName) {
 				opts.ReadFileFrom = ReadFromPatchDiff
 			}
+		} else {
```

Comment:
I think you want this else to line up with the if statement at 572; here you're setting ReadFromLocal to ReadFromPatch, for example

Reply:
yes